javascript/javascript2/week3/question3.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
+      #time-input {
+        padding: 10px;
+        font-size: 18px;
+        margin-right: 10px;
+      }
+
+      #start-button {
+        padding: 10px 20px;
+        font-size: 18px;
+        background-color: #4caf50;
+        color: white;
+        border: none;
+        cursor: pointer;
+      }
+
+      #feedback {
+        font-size: 20px;
+        font-weight: bold;
+        margin-top: 20px;
+      }
+    </style>
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Fastest Presser</h1>
+    <input type="text" id="time-input" />
+    <button id="start-button">Start game</button>
+    <div id="feedback"></div>
+
+    <script>
+      let countS = 0;
+      let countL = 0;
+      let time;
+      let timeoutId;
+
+      document
+        .getElementById("start-button")
+        .addEventListener("click", function () {
+          time = document.getElementById("time-input").value;
+          timeoutId = setTimeout(endGame, time * 1000);
+          document.addEventListener("keydown", countKeyPress);
+        });
+
+      function countKeyPress(event) {
```

Comment:
All works nicely here! Good work in the code, what can be imporved a bit is the user experience. 
Now we press the keys and we do not know if anything happens. It would be nice to print out the number of clicks every time we click and add a short instrucion what we are suppose to press :)